### PR TITLE
bugfix: persistent storage of sandbox metadata

### LINKED ISSUE #1164: [bug] The pod status data which get from CRI  is error
### Ⅰ. Issue Description
After pod created, if i restart pouch daemon, the sanboxMeta data store in SandboxStore will lost. Now if i get pod status from cri ,it will always get error.

### Ⅱ. Describe what happened

image

pouch version
GitCommit:       unknown
GoVersion:       go1.9.1
KernelVersion:   3.10.0-327.ali2012.alios7.x86_64
Os:              linux
Version:         0.4.0-dev
APIVersion:      1.24
Arch:            amd64
BuildTime:       unknown

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: YaoZengzeng <yaozengzeng@zju.edu.cn>

### Ⅰ. Describe what this PR did

fixes #1164

### CODE REVIEW

File: daemon/mgr/cri.go [hunk 1]
```diff
@@ -312,7 +327,10 @@ func (c *CriManager) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 		return nil, fmt.Errorf("failed to remove root directory %q: %v", sandboxRootDir, err)
 	}
 
-	c.SandboxStore.Remove(podSandboxID)
+	err = c.SandboxStore.Remove(podSandboxID)
+	if err != nil {
```
Comment (HusterWan) [inline daemon/mgr/cri.go]: Should we return err info to Caller, if we do not deal with the error, there will have more and more dirty data, WDYT?

**Comment by YaoZengzeng** [comment]
@allencloud PTAL

File: daemon/mgr/cri.go (hunk 1 above)
Comment (YaoZengzeng) [reply daemon/mgr/cri.go]: Agree :)

**Comment by allencloud** [comment]
LGTM

**Pull request merged**